packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -40,17 +40,66 @@ class PlatformCameraPosition {
 
 /// Pigeon representation of a CameraUpdate.
 class PlatformCameraUpdate {
-  PlatformCameraUpdate(this.json);
-
-  /// The update data, as JSON. This should only be set from
-  /// CameraUpdate.toJson, and the native code must interpret it according to the
-  /// internal implementation details of the CameraUpdate class.
-  // TODO(stuartmorgan): Update the google_maps_platform_interface CameraUpdate
-  //  class to provide a structured representation of an update. Currently it
-  //  uses JSON as its only state, so there is no way to preserve structure.
-  //  This wrapper class exists as a placeholder for now to at least provide
-  //  type safety in the top-level call's arguments.
-  final Object json;
+  PlatformCameraUpdate({required this.cameraUpdate});
```

Comment:
Making this a named argument vs a positional argument doesn't seem to add any clarity, and it makes the already-verbose code using this class even more verbose. Having the single argument just be positional would be clearer in this case IMO.

Reply:
As far as I can tell, Pigeon will convert positional arguments to named arguments in the output file, anyway. I tried making it positional in `messages.dart`, but after running Pigeon, the generated code does not change. Do you think it would still be useful for this file to use a positional argument?

Reply:
Oh, for some I thought we preserved the argument structure on the Dart side. In that case it doesn't matter.